src/isIsogram.test.js

```diff
@@ -6,4 +6,26 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
```

Comment:
Consider adding a test case for an empty string, as the task description specifies that an empty string should be considered an isogram. This will ensure that the function handles this edge case correctly.